source/transfer_list.rst

```diff
@@ -657,5 +659,103 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM event log table entry layout (XFERLIST_EVLOG)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This entry type holds TPM-related information for a platform. The TPM event log
+info is a region containing a TPM event log as defined by TCG EFI Protocol
+Specification [TCG_EFI]_.
+
+.. _tab_tpm_evlog:
+.. list-table:: TPM event log type layout
+   :widths: 2 2 4 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size of the event log in bytes + sizeof(flags) i.e. 0x4.
+
+   * - flags
+     - 0x4
+     - hdr_size
+     - flags are intended to signal properties of this TE. Bit 0 is
+       need_to_replay flag. Some firmware components may compute measurements
+       to be extended into a TPM and add them to the TPM event log, but those
+       components are unable  to access the TPM themselves. In this case, the
+       component should set the "need_to_replay" flag so that the next
+       component in the boot chain is aware that the TPM has not been updated.
+       A component with access to the TPM would replay the event log by reading
+       each measurement recorded and extending it into the TPM. Once the
+       measurements are extended into the TPM, then the "need_to_replay" flag
+       must be cleared if the transfer list is passed to additional firmware
+       components, means bootloader needs to replay the EventLog. Default
```

Comment:
Why not clear it unconditionally?

Reply:
So if the earlier boot stage doesn't set it, it will be clear only. The component which extend the PCR should only clear it.

Reply:
This sentence doesn't make sense: "Once the measurements are extended into the TPM, then the "need_to_replay" flag must be cleared if the transfer list is passed to additional firmware components, means bootloader needs to replay the EventLog.". It looks like the last clause ("means bootloader needs to replay the EventLog") should be removed.